pyrobosim/pyrobosim/gui/world_canvas.py

```diff
@@ -171,6 +171,17 @@ def __init__(
             self.nav_animator.timeout.connect(self.nav_animation_callback)
             self.nav_animator.start(sleep_time_msec)
 
+    def toggle_collision_polygons(self):
+        """Shows/hides collision polygons."""
+        print(
+            "Disabling collision polygons"
+            if self.show_collision_polygons
+            else "Enabling collision polygons"
+        )
```

Comment:
I think it's clearer if you move this print after the toggle, so that the `if` shows Enabling and the `else` shows Disabling.

Reply:
Done! Thank you so much for the pointers